tests/foreman/ui/test_organization.py

```diff
@@ -199,7 +199,7 @@ def test_positive_create_with_all_users(session, module_target_sat):
 
     :expectedresults: Organization and user entities assigned to each other
 
-    :BZ: 1321543
+    :verifies: SAT-25386
```

Comment:
```suggestion
    :Verifies: SAT-25386
```
@jyejare can we keep the token consistent? let's use `Verifies`.